python/rmm/_lib/device_buffer.pyx

```diff
@@ -211,6 +211,13 @@ cpdef void copy_ptr_to_host(uintptr_t db,
     hb : ``bytes``-like buffer to write into
     stream : CUDA stream to use for copying, default 0
 
+    Note
+    ----
+
+    If ``stream`` is the default stream, we synchronize on it.
```

Comment:
Please don't use the royal we in docs. The queen doesn't synchronize, this function does.  Also please be explicit about *when* synchronization is done (before the copy? after? both?)

Reply:
Ok have made some changes in recent commits. Please take a look 🙂